oscar_odin/mappings/address.py

```diff
@@ -14,22 +11,31 @@
 ShippingAddressModel = get_model("order", "ShippingAddress")
 CountryModel = get_model("address", "Country")
 
+# mappings
+OscarBaseMapping = get_class("oscar_odin.mappings._common", "OscarBaseMapping")
```

Comment:
Since there are some breaking changes in this PR, we should also consider renaming this _common file. OscarBaseMapping is not just used internally and is being used in projects importing resources other than products. Same is the case with _model_mapper.